news/2018-08-26-project-loved-week-of-august-26th.md

```diff
@@ -0,0 +1,214 @@
+---
+layout: post
+title: "Project Loved: Week of August 26th"
+date: 2018-08-26 21:00:00 +0000
+---
+
+[Project Loved](https://osu.ppy.sh/community/forums/120) is back, this time with *twice* the amount of osu!standard maps. Check out our picks, and vote for the maps you want to see Loved!
+
+[![](/wiki/shared/news/banners/project-loved.jpg)](https://osu.ppy.sh/community/forums/120)
+
+Welcome back to [Project Loved](https://osu.ppy.sh/community/forums/120)! Every two weeks, we pick out a few beatmaps from each game mode that will get their shot at making it into the Loved category. From the maps we selected, you can now vote whether or not you want them to be Loved! Click on each maps' header image in this post to be taken to their voting threads.
+
+Following [-Kazu-](https://osu.ppy.sh/users/920861)'s recruitment onto the osu!taiko Loved captain team last week, [Gezoda](https://osu.ppy.sh/users/481582) has decided to take his leave. He'll still be helping out with management of our Google sheets, but he won't be picking the maps anymore. Thanks for all of the work you've put into this project, Gezo!
```

Comment:
```diff
- our Google sheets, but
+ our Google Sheets, but
```

/edit from just reading ahead (line 30), probably doesn't matter as much since other occurrences are similar (i.e. Google forms and Google sheets)